jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -59,6 +84,8 @@
 
     @Nullable @Parameter private String credHelper;
 
+    @Parameter private AuthConfiguration auth = new AuthConfiguration();
```

Comment:
If the below are final, this could be final too?

Reply:
Oh, I think I meant to change those back.